public/_posts/_articles/yearn-finance/yvboost/es.md

```diff
@@ -0,0 +1,99 @@
+---
+title:  "Más impulsos, compuestos y mojitos para ti con yvBOOST"
+image:
+  src: ./cover.jpg
+  width: 1050
+  height: 543
+date: '2021-09-14'
+author: Qosmonot
+translator: RestlessMik3
+---
+
+# Más impulsos, compuestos y mojitos para ti con yvBOOST
+
+### La relación de colaboración de Yearn con [Curve.fi](http://curve.fi/) es fundamental para nuestra infraestructura y fundamental para impulsar tus impulsos.
+
+🔵🤝🌈
+
+_publicación escrita por_ [_qosmonot_](http://twitter.com/qosmonot), un colaborador de Yearn
+
+Muchas vaults de Yearn tienen estrategias basadas en los grupos de Curve y, por lo tanto, usan el token CRV para optimizarlas votando para dirigir las asignaciones de recompensas hacia esos grupos. Yearn ha estado aumentando constantemente su titularidad de tokens CRV para proporcionar recompensas cada vez mayores para esas vaults.
+
+![](image1.jpg?w=1050&h=651)
+
+Muchas vaults de Yearn tienen estrategias basadas en los grupos de Curve y, por lo tanto, usan el token CRV para optimizarlas votando para dirigir las asignaciones de recompensas hacia esos grupos. Yearn ha estado aumentando constantemente su titularidad de tokens CRV para proporcionar recompensas cada vez mayores para esas vaults.
```

Comment:
"Muchas vaults de Yearn tienen estrategias basadas en los grupos de Curve y, por lo tanto, usan el token CRV para optimizarlas votando para dirigir las asignaciones de recompensas hacia esos grupos. Yearn ha estado aumentando constantemente su titularidad de tokens CRV para proporcionar recompensas cada vez mayores para esas vaults."

Isn't this whole thing repeated?

Reply:
You mean in the same paragraph? first part talks about what Yearn does, second part about Yearn accumulating more CRV for what it does.